### ISSUE TITLE
Range-attack units cannot capture enemy workers

### ISSUE THREAD

**Issue by Kzer-Za** [yairm210/Unciv#12541]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Game Version

4.14.4

### Describe the bug

There does not seem to be a way to capture an enemy worker with a military unit that has a ranged attack (at least on Android). In Civ 5 per se it is possible.

### Steps to Reproduce

Prerequisites:

1. Be in war with an opponent.
2. Have your ranged military unit, such as archer, keshik, etc near an enemy worker.

Steps:

1. Select your ranged unit.
2. Tap on the enemy worker.

The actual result:

The ranged units attacks and eventually kills the worker.

The expected result:

To have an option to capture the worker, since it is possible to do it in vanilla Civ 5.

### Operating System

Android

**Comment by metablaster** [comment]
It would be awesome to be able to capture civilian units with ranged ones instead of attacking them by default.

**Comment by SpacedOutChicken** [comment]
In the latest version of Unciv, you can capture enemy workers with ranged units. Instead of clicking the attack button, you tell the unit to move onto the worker by selecting the spot where the worker is, and it will make that move instead of making the ranged attack. If you update to the latest version, this problem should go away.

**Comment by metablaster** [comment]
@SpacedOutChicken 
Thank you for letting us know!
Maybe this should be explained in Civilopedia?

**Comment by SHypoleac** [comment]
> @metablaster  Maybe this should be explained in Civilopedia?

Nope, this is the biggest secret of this game, shared only with worthy contributors on this site ;)

**Comment by yairm210** [comment]
Closing as "actually works"
